File: model/usuario.php
<?php

class Usuario{

    private $idUsuario;
    private $nome;
    private $matricula;
    private $empresa;
    private $cargo;
    private $login;
    private $senha;
    private $statusUsuario;

    function __construct($idUsuario, $nome, $matricula, $empresa, $cargo, $login, $senha, $statusUsuario){
        $this->setIdUsuario($idUsuario);
        $this->setNome($nome);
        $this->setMatricula($matricula);
        $this->setEmpresa($empresa);
        $this->setCargo($cargo);
        $this->setLogin($login);
        $this->setSenha($senha);
        $this->setStatusUsuario($statusUsuario);
    }

    function setIdUsuario($idUsuario){
        $this->idUsuario = $idUsuario;
    }

    function setNome($nome){
        $this->nome = $nome;
    }

    function setMatricula($matricula){
        $this->matricula = $matricula;
    }

    function setEmpresa($empresa){
        $this->empresa = $empresa;
    }

    function setCargo($cargo){
        $this->cargo = $cargo;
    }

    function setLogin($login){
        $this->login = $login;
    }

    function setSenha($senha){
        $hashPassword = password_hash($senha, PASSWORD_DEFAULT);
        $this->senha = $hashPassword;
    }

    function setStatusUsuario($status){
        $this->statusUsuario = $status;
    }

    function getIdUsuario(){
        return $this->idUsuario;
    }

    function getNome(){
        return $this->nome;
    }

    function getMatricula(){
        return $this->matricula;
    }

    function getEmpresa(){
        return $this->empresa;
    }

    function getCargo(){
        return $this->cargo;
    }

    function getLogin(){
        return $this->login;
    } 

    function getSenha(){
        return $this->senha;
    }

    function getStatusUsuario(){
        return $this->statusUsuario;
    }
    
    function __toString(){
        return 
        "<br>ID Usuario: ".$this->getIdUsuario().
        "<br>Nome: ".$this->getNome().
        "<br>Matricula: ".$this->getMatricula().
        "<br>FK empresa: ".$this->getEmpresa().
        "<br>FK cargo: ".$this->getCargo().
        "<br>Login: ".$this->getLogin().
        "<br>Hash da senha: ".$this->getSenha().
        "<br>Flag de status do usuario: ".$this->getStatusUsuario()."<br>"; 
    }
}

?>

File: aplicativo/app/src/main/java/com/example/echeckin/model/Usuario.java
package com.example.echeckin.model;

public class Usuario {

    private int idUsuario;
    private String nomeUsuario;
    private int fkCargoUsuario;
    private String login;
    private String senha;
    private int statusUsuario;

    public void Usuario(int idUsuario, String nomeUsuario, int fkCargoUsuario, String login, String senha, int statusUsuario){
        this.setIdUsuario(idUsuario);
        this.setNomeUsuario(nomeUsuario);
        this.setFkCargoUsuario(fkCargoUsuario);
        this.setLogin(login);
        this.setSenha(senha);
        this.setStatusUsuario(statusUsuario);
    }

    public void setIdUsuario(int idUsuario){
        this.idUsuario = idUsuario;
    }

    public void setNomeUsuario(String nomeUsuario){
        this.nomeUsuario = nomeUsuario;
    }

    public void setFkCargoUsuario(int fkCargoUsuario){
        this.fkCargoUsuario = fkCargoUsuario;
    }

    public void setLogin(String login){
        this.login = login;
    }

    public void setSenha(String senha){
        this.senha = senha;
    }

    public void setStatusUsuario(int statusUsuario){
        this.statusUsuario = statusUsuario;
    }

    public int getIdUsuario(){
        return this.idUsuario;
    }

    public String nomeUsuario(){
        return this.nomeUsuario;
    }

    public int getFkCargoUsuario(){
        return this.fkCargoUsuario;
    }

    public int getStatusUsuario(){
        return this.statusUsuario;
    }

}
